src/tests/unit/tests/injected/visualization/svg-drawer.test.ts

```diff
@@ -891,10 +901,10 @@ describe('SVGDrawer', () => {
 
         drawerUtilsMock.verifyAll();
 
-        expect(circles.length).toBe(3);
+        expect(circles.length).toBe(4);
```

Comment:
@lisli1 I just merged my e2e tests PR that also touches this test so it's probably worthwhile to merge main into this branch to make sure these assertions are still correct now.